modules/osdk-prepare-supported-images.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:type: Ansible
+:type_lc: ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:type: Helm
+:type_lc: helm
+endif::[]
+
+:_content-type: PROCEDURE
+[id="osdk-prepare-supported-images_{context}"]
+= Optional: Using supported images with upstream versions of the Operator SDK
+
+If you use an upstream version of the Operator SDK to have access to newer features and bug fixes, update your {type}-based Operator project to use supported images.
```

Comment:
Can we link the "upstream version of the Operator SDK" to the upstream SDK doc so readers can easier tell what does "upstream" vs "downstream" mean?

Thinking of something like:

If you use an [upstream version of the Operator SDK](https://sdk.operatorframework.io/docs/installation/) to have access to newer features and bug fixes, update your